src/AppInstallerCommonCore/Versions.cpp

```diff
@@ -42,6 +48,26 @@ namespace AppInstaller::Utility
 
     bool Version::operator<(const Version& other) const

```

Comment:
>< [](start = 26, length = 1)

should we sort Unknown < Latest? Looks like currently Latest < Unknown?

Reply:
Added a test to ensure Unknown < Latest, currently passing.

---
In reply to: [507387816](https://github.com/microsoft/winget-cli/pull/598#discussion_r507387816) [](ancestors = 507387816)